src/main/java/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event extends Task {
+
```

Comment:
Again, please try to be consistent with your empty lines, you could consider removing it